### ISSUE TITLE
Create Firmware Download Executor

### ISSUE THREAD

**Issue by FendVario1** [amosproj/amos2022ss01-firmware-downloader#34]
## User story
1. As a User
2. I want / need a python program
3. So that I can just start the program and all firmware extraction modules get executed automatically.

## Acceptance criteria
* Setup of a basic Python program
* Ability to call all vendor submodules
* Should not break if individual vendor module fails; all other modules should be executed
* Documentation of Module requirements (what is needed for the executor to run module)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
